xwiki-platform-core/xwiki-platform-user/xwiki-platform-user-api/src/main/java/org/xwiki/user/UserManager.java

```diff
@@ -38,4 +40,16 @@ public interface UserManager
      *     the user
      */
     boolean exists(UserReference userReference) throws UserException;
+
+    /**
+     * @param wiki the reference of the wiki where to check of users exist
+     * @return true if at least one user is available on that target wiki
```

Comment:
I'd be a bit more specific in the javadoc by specifying we're looking if the wiki contains a local user. A global user can always be available...

Reply:
"contains" felt explicit enough to me. But sure, let's make extra sure it's clear that it's only in the target wiki.